.storybook/components/FluentDocsPage.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
```

Comment:
This component was copied over from https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx and will be removed once original component will be published as a part of a new package.